### PR TITLE
Update edit and addward command description in UG and add a help.txt file

### PR DESCRIPTION
**Pull request by KSunil2001**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -94,15 +94,22 @@ Shows a list of all patients with their details in the system.
...
+- The status of a patient is either `GRAY` or `GREEN` or `YELLOW` or `RED`.
+- The ward allocated to a patient is represented as an uppercase alphabet followed by 2 digits. E.g `A01`.
+- The discharge date-time is of the `dd/MM/yyyy HHmm` format. E.g `12/03/2023 1200` is interpreted as 12th March 2023 1200hrs.
+
 Examples:
 
-- `edit 1 s/GREEN` Edits the status of the first currently displayed patient with to be `GREEN`.
+- `edit 1 s/GREEN` Edits the status of the first currently displayed patient to be `GREEN`.
+- `edit 5 w/A01` Edits the ward of the fifth currently displayed patient to be `A01`. 
+- `edit 4 d/27/07/2023 1600` Edits the discharge date-time of the fourth currently displayed patient to be `27/07/2023 1600` which is read as 27th July 2023 1600hrs.
```
Comment (yitong241) [inline docs/UserGuide.md:112]: clear and precise use of words, good job 😀

**Review by yitong241 — APPROVED** [review]
Neat and nice UG, nice work 🥰

File: docs/UserGuide.md [hunk 2]
```diff
@@ -94,15 +94,22 @@ Shows a list of all patients with their details in the system.
...
+### Editing a patient’s details in the system: `edit`
 
-Edit an existing patient’s condition.
+Edit an existing patient’s status or ward or discharge date-time.
 
 Format: `edit INDEX [s/STATUS] [w/WARD] [d/DISCHARGE]​`
 
+- Edits the patient's details at the specified index as of the currently displayed list.
+- The status of a patient is either `GRAY` or `GREEN` or `YELLOW` or `RED`.
+- The ward allocated to a patient is represented as an uppercase alphabet followed by 2 digits. E.g `A01`.
```
Comment (nramapurath) [inline docs/UserGuide.md:105]: Should we restrict it to one uppercase alphabet followed by 2 digits? Considering that we allow ward to be any alphanumeric string..

File: docs/UserGuide.md [hunk 3]
```diff
@@ -137,18 +144,21 @@ Examples:
 
 ### Adding a ward to the system: `addward`
 
-Adds the patient (NRIC, name and condition).
+Adds the ward (name and capacity).
 
 Format: `addward name/NAME [c/CAPACITY]​`
 
+- The ward name is represented as an uppercase alphabet followed by 2 digits. E.g `A01`.
```
Comment (nramapurath) [inline docs/UserGuide.md:151]: Related to my previous comment on ward naming

File: src/main/resources/help.txt [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+   Note: Capacity is optional and its default value is 10.
+
+8. Sorting patients: sort
+
+   Sorts the patients by name or status or ward or discharge date in either ascending or descending order in the system.
+   Format 1: sort name/ ORDER
+   Format 2: sort s/ ORDER 
+   Format 3: sort w/ ORDER
+   Format 4: sort d/ ORDER
+   Example: sort name/ ASC
```
Comment (nramapurath) [inline src/main/resources/help.txt:77]: Would it be better without the space between the keyword and ORDER? So as to standardize with the way the other commands have been listed

**Pull request merged**